packages/sakuli-legacy/src/loader/legacy-project-properties.class.ts

```diff
@@ -166,4 +166,15 @@ export class LegacyProjectProperties {
     @StringProperty('sakuli.screenshot.storage')
     screenshotStorage: ScreenshotFolderStructure = "hierarchical";
 
+    /**
+     * Delay between mouse actions e.g. mouse button down, mouse button up, scroll, etc. in ms
+     */
+    @NumberProperty('sakuli.mouse.action.delay')
+    mouseActionDelay: number = 200;
```

Comment:
Since the `sikuliClickDelay` property has not been applied at all until now (went unnoticed, my bad), we used the default timeout of 10ms.
Changing this to 200ms makes me think about possible problems when performing e.g. a double-click?

Reply:
Yeah you're right. I'll reduce it as suggested.